libs/Microsoft.MixedReality.WebRTC/DataChannel.cs

```diff
@@ -146,25 +160,9 @@ internal DataChannel(IntPtr nativeHandle, PeerConnection peer, IntPtr argsRef, i
         }
 
         /// <summary>
-        /// Finalizer to ensure the data track is removed from the peer connection
-        /// and the managed resources are cleaned-up.
-        /// </summary>
-        ~DataChannel()
-        {
-            Dispose();
-        }
-
-        /// <summary>
-        /// Remove the data track from the peer connection and destroy it.
+        /// Dispose of the native data channel. Invoked by its owner (<see cref="PeerConnection"/>).
         /// </summary>
-        public void Dispose()
-        {
-            State = ChannelState.Closing;
-            // This will invoke some callback from native, which calls Destroy()
-            PeerConnection.RemoveDataChannel(_nativeHandle);
-        }
-
-        internal void Destroy()
+        internal void DestroyNative()
```

Comment:
The call to GC.SuppressFinalize is unnecessary now, right?

Reply:
Fixed.